src/triedbmut.rs

```diff
@@ -67,16 +71,24 @@ impl<'a> TrieDBMut<'a> {
     ) -> crate::Result<H256> {
         match cur_node_hash {
             Some(hash) => {
-                let node_rlp = self.db.get(&hash).ok_or_else(|| TrieError::IncompleteDatabase(hash))?;
-
-                match RlpNode::decoded(&node_rlp) {
+                let node_rlp;
```

Comment:
There is no change in performance when writing the value to the trie, so I will exclude the cache from this.

Reply:
A write-through cache doesn't increase writing performance, but it improves reading performance.
Please add the test case that reads data after inserting data.
It will show how much the write-through cache has improved writing performance.